python/private/pypi/pep508_evaluate.bzl

```diff
@@ -344,6 +344,14 @@ def _env_expr(left, op, right):
         return left in right
     elif op == "not in":
         return left not in right
+    elif op == "<":
+        return left < right
+    elif op == "<=":
+        return left <= right
+    elif op == ">":
+        return left > right
+    elif op == ">=":
+        return left >= right
```

Comment:
Actually, could you please add tests to `//tests/pypi/pep508:evaluate_tests.bzl` to document what this behaviour is?

I know it will be very tedious, but having tests that cover all branches is a good form of semi-documentation.

Reply:
Done.

(Was fairly easy once you pointed me to the test file, thanks!)